src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1125,22 +1484,35 @@ public enum RisingRivals implements LogicCardInfo {
           pokeBody "Frost Wind", {
             text "As long as Glaceon is your Active Pokémon, any damage done to your Pokémon by your opponent’s attacks is reduced by 10 ."
             delayedA {
+              before APPLY_ATTACK_DAMAGES, {
+                bg.dm().each {
+                  if(self.active && it.to.owner == self.owner && it.from.owner == self.owner.opposite && it.dmg.value && it.notNoEffect){
+                    bc "Frost Wind -10"
```

Comment:
I really recommend using $thisAbility and $thisMove where appropriate. Makes it easier to copy and paste when relevant.